dev/tasks/tasks.yml

```diff
@@ -1033,6 +1033,14 @@ tasks:
       flags: '-e ARROW_SOURCE_HOME="/arrow" -e FORCE_BUNDLED_BUILD=TRUE -e LIBARROW_BUILD=TRUE -e ARROW_DEPENDENCY_SOURCE=SYSTEM'
       image: ubuntu-r-only-r
 
+  test-r-offline-minimal:
+      ci: azure
+      template: r/azure.linux.yml
+      params:
+        r_org: rocker
+        r_image: r-base
+        r_tag: latest
+        flags: '-e TEST_OFFLINE_BUILD=true'
```

Comment:
I put this one on azure because the one above was on azure, but feel free to change to a different platform.

Reply:
Azure is fine for this one. TBH, I picked Github Actions for the maximal build out of convenience since we already have a model that has dependent steps. But our CI system (AKA crossbow) is designed to be spread across a number of systems like this, so it's totally fine to use two different services for these two jobs.